### PR TITLE
fix(feedback): hide API from public until release

### PR DESCRIPTION
**Pull request by armcknight**
These were added to the public header in #4284 but should not have been exposed before the initial rollout of the feature. Let's move them into private headers until we're ready.

#skip-changelog

### CODE REVIEW

**Comment by armcknight** [comment]
The lint failures are bubbling up from an upstream PR: https://github.com/getsentry/sentry-cocoa/pull/4630#issuecomment-2540323633

**Review by brustolin — APPROVED** [review]

File: Sources/Sentry/Public/SentrySDK.h [hunk 1]
```diff
@@ -6,11 +6,18 @@
 
 @protocol SentrySpan;
 
-@class SentryOptions, SentryEvent, SentryBreadcrumb, SentryScope, SentryUser, SentryId,
-    SentryUserFeedback, SentryTransactionContext;
+@class SentryBreadcrumb;
+@class SentryEvent;
+@class SentryFeedback;
+@class SentryId;
```
Comment (philipphofmann) [inline Sources/Sentry/Public/SentrySDK.h:12]: Thanks 🥇

**Review by philipphofmann — APPROVED** [review]
LGTM

File: Sources/Sentry/Public/SentrySDK.h (hunk 1 above)
Comment (armcknight) [reply Sources/Sentry/Public/SentrySDK.h:12]: heh i didn't even mean for this to go in here, i think.

File: Samples/iOS-Swift/iOS-Swift.xcodeproj/xcshareddata/xcschemes/iOS-Swift.xcscheme [hunk 2]
```diff
@@ -69,6 +69,10 @@
          </BuildableReference>
       </BuildableProductRunnable>
       <CommandLineArguments>
+         <CommandLineArgument
+            argument = "--io.sentry.disable-everything"
+            isEnabled = "NO">
+         </CommandLineArgument>
```
Comment (armcknight) [inline Samples/iOS-Swift/iOS-Swift.xcodeproj/xcshareddata/xcschemes/iOS-Swift.xcscheme:75]:
```suggestion
```

File: Sources/Sentry/Public/SentrySDK.h (hunk 1 above)
Comment (armcknight) [inline Sources/Sentry/Public/SentrySDK.h:12]:
```suggestion
@class SentryOptions, SentryEvent, SentryBreadcrumb, SentryScope, SentryUser, SentryId,
     SentryUserFeedback, SentryTransactionContext;
```

File: Sources/Sentry/Public/SentrySDK.h [hunk 3]
```diff
@@ -6,11 +6,18 @@
...
 @class SentryMetricsAPI;
-@class UIView;
+@class SentryOptions;
 @class SentryReplayApi;
+@class SentryScope;
+@class SentryTransactionContext;
+@class SentryUser;
+@class SentryUserFeedback;
+@class UIView;
```
Comment (armcknight) [inline Sources/Sentry/Public/SentrySDK.h:20]:
```suggestion
@class UIView;
```

File: Sources/Sentry/include/SentryPrivate.h [hunk 4]
```diff
@@ -4,6 +4,8 @@
 #import "SentryDispatchQueueWrapper.h"
 #import "SentryNSDataUtils.h"
 #import "SentryRandom.h"
+#import "SentrySDK+Private.h"
+#import "SentryScope+Private.h"
```
Comment (armcknight) [inline Sources/Sentry/include/SentryPrivate.h:8]:
```suggestion
```

**Pull request merged**